Makefile

```diff
@@ -81,7 +79,8 @@ qvm-ng: system-index.txt
 		--load build-app-ng.lisp \
 		$(FOREST_SDK_OPTION)
 
-qvm-sdk-base: FOREST_SDK_FEATURE=--eval '(pushnew :forest-sdk *features*)'
+qvm-sdk-base: FOREST_SDK_FEATURE=--eval '(pushnew :forest-sdk *features*)' \
+	--eval '(push :drakma-no-ssl *features*)'
```

Comment:
Purrrhaps this should also be `pushnew` for pleasing symmetry with the `:forest-sdk` feature

Reply:
Never happy are you?

Reply:
![grumpy-cat](https://user-images.githubusercontent.com/86076/73776512-7b9a8700-474d-11ea-9796-4ba9abf89980.jpg)